ui/components/component-library/avatar-account/avatar-account.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import classnames from 'classnames';
+import Jazzicon from '../../ui/jazzicon/jazzicon.component';
+import BlockieIdenticon from '../../ui/identicon/blockieIdenticon/blockieIdenticon.component';
+
+import type { PolymorphicRef, BoxProps } from '../box';
+
+import { AvatarBase } from '../avatar-base';
+import {
+  AvatarAccountDiameter,
+  AvatarAccountVariant,
+  AvatarAccountSize,
+  AvatarAccountComponent,
+  AvatarAccountProps,
+} from './avatar-account.types';
+
+export const AvatarAccount: AvatarAccountComponent = React.forwardRef(
+  <C extends React.ElementType = 'div'>(
+    {
+      size = AvatarAccountSize.Md,
+      address,
+      className = '',
+      variant = AvatarAccountVariant.Jazzicon,
+      ...props
+    }: AvatarAccountProps<C>,
+    ref?: PolymorphicRef<C>,
+  ) => (
+    <AvatarBase
+      ref={ref}
+      size={size}
```

Comment:
as this component uses `AvatarBase` as base, it does not accept 
`size = AvatarAccountSize.Md` so we either pass `size = AvatarBaseSize.Md` or make changes to AvatarBase interface `size?: AvatarBaseSize | AvatarAccountSize;` 

![image](https://github.com/MetaMask/metamask-extension/assets/79097544/01aa0325-7de8-4ef3-9cf3-13d952b8b955)

Reply:
> See the suggestion I made in the types file and that should hopefully fix this.
> 
> You can always reference the code from `avatar-token` or `avatar-network` since those are now up to date.

Thank you for the suggestions it fixed the errors :)